python/pyarrow/tests/test_pandas.py

```diff
@@ -3941,3 +3945,28 @@ def test_metadata_compat_missing_field_name():
     result = table.to_pandas()
     # on python 3.5 the column order can differ -> adding check_like=True
     tm.assert_frame_equal(result, expected, check_like=True)
+
+
+@pytest.mark.skipif(pq is None, reason="Parquet not available")
```

Comment:
```suggestion
@pytest.mark.parquet
```

(the mark will automatically ensure the tests are skipped as well)

Reply:
Fixed.